curriculum/challenges/english/25-front-end-development/lab-user-configuration-manager/684aaf9ec670c68d20efd0d0.md

```diff
@@ -0,0 +1,384 @@
+---
+id: 684aaf9ec670c68d20efd0d0
+title: Build a User Configuration Manager
+challengeType: 27
+dashedName: build-a-user-configuration-manager
+---
+
+# --description--
+
+Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should create a dictionary named `settings` that would store the following user configuration preferences in this format:
```

Comment:
I feel like it needs to be clear that this dictionary is only here for testing purposes. 
Because looking at the final solution, this is added at the end only to test out the function.
maybe renaming it would be good. so something like `test_settings`